synfig-studio/src/synfigapp/actions/spine_export.h

```diff
@@ -0,0 +1,63 @@
+/* === S Y N F I G ========================================================= */
+/*! \file spine_export.h
+** \brief Spine JSON Export Header File
+**
+** This file is part of Synfig.
+**
+** Synfig is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 2 of the License, or
+** (at your option) any later version.
+**
+** Synfig is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+** \endlegal
+*/
+/* ========================================================================= */
+
+#ifndef __SYNFIG_APP_ACTION_SPINEEXPORT_H
+#define __SYNFIG_APP_ACTION_SPINEEXPORT_H
```

Comment:
Lots of synfig code use this #include guard naming convention, but `__`-prefix is meant to be reserved to c++ language.
https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier/228797#228797